<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>641</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>gestion de voyageurs</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>591</width>
        <height>471</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ajouter voyageur</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouter voyageur</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Pr√©nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_cin1">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Telephone</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_telephone">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="ajoutervoyageur">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>200</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Abdelmoumen Djabu/Ajouter.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>groupBox</zorder>
       <zorder>label_10</zorder>
       <zorder>ajoutervoyageur</zorder>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Afficher voyageur</string>
       </attribute>
       <widget class="QTableView" name="tabvoyageur">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>101</y>
          <width>511</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>chercher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Veuillez saisir le CIN</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>supprimer voyageur</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>40</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer voyageur</string>
        </property>
        <widget class="QLabel" name="label_19">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_cin_5">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>120</y>
           <width>161</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Abdelmoumen Djabu/supprimer.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimervoyageur">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>120</y>
           <width>151</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>modifier voyageur</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier voyageur</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Pr√©nom</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Cin</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_nom_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_prenom_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_cin_3">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>61</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Telephone</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_telephone_2">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>200</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Abdelmoumen Djabu/Modifier.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>groupBox_2</zorder>
       <zorder>label_12</zorder>
       <zorder>pushButton_2</zorder>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Gestion des billets</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ajouter billet</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Ajouterr billet</string>
        </property>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>destination</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_destination_billet">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_code_billet">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_18">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>date</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_date_billet">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_22">
         <property name="geometry">
          <rect>
           <x>-70</x>
           <y>150</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>E:/Abdelmoumen Djabu/tri.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>161</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>210</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Abdelmoumen Djabu/Ajouter.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ajouterbillet">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>210</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(80, 179, 255);
background-color: rgb(255, 0, 0);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Afficher billet</string>
       </attribute>
       <widget class="QTableView" name="tabbillet">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>291</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>220</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>../Abdelmoumen Djabu/tri.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>tabbillet</zorder>
       <zorder>label_25</zorder>
       <zorder>pushButton_5</zorder>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>supprimer billet</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Supprimer billet</string>
        </property>
        <widget class="QLabel" name="label_20">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_code_">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>40</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_23">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>171</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Abdelmoumen Djabu/supprimer.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="supprimerbillet">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>120</y>
           <width>141</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>modifier billet</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>311</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Modifier billet</string>
        </property>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>91</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>destination</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_15">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="la_des">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="le_code">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>30</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_16">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>date</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="la_date">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>60</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_24">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>120</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>../Abdelmoumen Djabu/Modifier.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>120</y>
           <width>121</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Abdelmoumen Djabu/BAckground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
